protobuf.bzl

```diff
@@ -506,3 +514,27 @@ def check_protobuf_required_bazel_version():
     copied filegroup. (Fixed in bazel 0.5.4)
     """
     versions.check(minimum_bazel_version = "0.5.4")
+
+
+def conformance_test(name, testee, language):
```

Comment:
Could we have it pass failure list files directly?  Passing language and inferring filenames from that feels a bit too indirect/magical.

Reply:
Yeah, I prefer your suggestion and reimplemented it. I mistakenly thought I couldn't pass args to sh_test, but turns out you can and it's much easier this way.